### ISSUE TITLE
Cannot drag player tscn file into game

### ISSUE THREAD

**Issue by StoreBoughtRocketGames** [godotengine/godot#68674]
### Godot version

4.0 dev beta 4 e6751549c

### System information

Windows 10, Vulkan, AMD - Radeon RX 5500 XT, 

### Issue description

After importing a new glb file of a model into my project, I copied over nodes from PlayerCharacter to Player (with the intent of adding a new animation). Suddenly, I'm unable to drag my player scene or PlayerCharacter scene into the editor or scene tree anymore.

### Steps to reproduce

Try to add the Player.tscn or PlayerCharacter.tscn file into the scene tree.

### Minimal reproduction project

https://drive.google.com/file/d/1swjsyWvbE1iCX7ANFRT2_BqK4DIuWt4l/view?usp=sharing

**Comment by timothyqiu** [comment]
File size is exactly the reason, the tscn file embeds a 200M+ array mesh.

The tscn file can be dragged into the scene eventually, but the editor is stuttering. Saving it as a separate file solves the stuttering.

**Comment by timothyqiu** [comment]
The array mesh has several textures embedded inside 🤣

**Comment by StoreBoughtRocketGames** [comment]
> The array mesh has several textures embedded inside 🤣

The file size of the exported glb file from blender is only 4MB. How does it end up ballooning so much when it gets imported to Godot? For reference, this same project in Godot 3.5 has no issues.

**Comment by Zireael07** [comment]
IIRC there was an issue about some weird embedding by mistake case, might have happened to you too

**Comment by KoBeWi** [comment]
When you open the model scene, you need to open is as inherited, not directly (you get a prompt when opening).
Can you attach the original glb file?

**Comment by StoreBoughtRocketGames** [comment]
@KoBeWi This isn't the original, but I was able to regenerate one from Blender. [LowpolyPlayerRigged.zip](https://github.com/godotengine/godot/files/10016441/LowpolyPlayerRigged.zip)

**Comment by KoBeWi** [comment]
Ok, I re-read the issue and you say you copied the nodes from the model scene. We have this dialog that says how to correctly handle models, did you see it?
![image](https://user-images.githubusercontent.com/2223172/202035845-9bce2d15-3b9c-47c1-9147-b88182467efd.png)

Seems like opening the scene directly is a common mistake, even though it's not obvious how to do it in 4.0, because double-click opens the import window.

**Comment by StoreBoughtRocketGames** [comment]
Yes, I opened the model with the Open Anyway button, and then saved it as a separate scene.

**Comment by Calinou** [comment]
Duplicate of https://github.com/godotengine/godot/issues/33379.
